config.toml

```diff
@@ -85,7 +85,9 @@ respectManualOfflineSoft=false
 debug = true
 
 # stdout: output is redirected to proxysql logs
-# file: output is written to the file pointed by logFile
+# file: output is written to the file pointed by logFile. When set to 'file',
+#       make sure the user proxysql:proxysql has anough permissions to write to
```

Comment:
enough? 
appropriate?
not sure what anough is.

Reply:
The intention here is to tell the users to set it to a location which the user proxysql:proxysql can access if he wants the logging to happen in a separate file. Can you please suggest better wordings to say the same?

Reply:
Thanks. Updated.